manim/animation/creation.py

```diff
@@ -200,6 +202,104 @@ def __init__(
         super().__init__(mobject, rate_func=rate_func, remover=remover, **kwargs)
 
 
+class Add(Animation):
+    """Adds a Mobject to the scene with no animated effect. This is functionally equivalent to :meth:`~.Scene.add`
+
+    Parameters
+    ----------
+    mobject : :class:`~.Mobject`
```

Comment:
Is this sufficient or should `OpenGLMobject` also be mentioned? If so, how? Same for `Remove`

Reply:
I think its better to mention them as we will be using OpenGL more in the upcoming updates

Reply:
We'll need to find a global solution for this dilemma eventually. I wouldn't explicitly mention OpenGLMobject here right now.